ocaml/xapi/create_storage.ml

```diff
@@ -41,17 +41,11 @@ let plug_all_pbds __context =
     my_pbds;
   !result
 
-let maybe_reenable_cluster_host __context =
-  let host = Helpers.get_localhost __context in
-  match Xapi_clustering.find_cluster_host ~__context ~host with
-  | Some self ->
-     Xapi_cluster_host.enable ~__context ~self
-  | None -> ()
 
 let plug_unplugged_pbds __context =
   (* If the plug is to succeed for SM's requiring a cluster stack
    * we have to enable the cluster stack too if we have one *)
-  log_and_ignore_exn(fun () -> maybe_reenable_cluster_host __context);
+  log_and_ignore_exn (fun () -> Xapi_cluster_host.resync_host ~__context ~host:Helpers.(get_localhost ~__context));
```

Comment:
This line should be deleted completely. We resync when we get the IP, and in cluster-pool-resync. Thats it.

Reply:
Looks like we forgot to delete this line

Reply:
Looks like we forgot to delete this line